tests/migration/Core/V6_7/Migration1756068709FixAddressNameFieldLengthsTest.php

```diff
@@ -0,0 +1,85 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Migration\Core\V6_7;
+
+use Doctrine\DBAL\Connection;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Framework\Test\TestCaseBase\KernelLifecycleManager;
+use Shopware\Core\Migration\V6_7\Migration1756068709FixCustomerAddressFirstNameLength;
+use Shopware\Core\Migration\V6_7\Migration1756068710FixCustomerAddressLastNameLength;
+use Shopware\Core\Migration\V6_7\Migration1756068711FixOrderAddressFirstNameLength;
+use Shopware\Core\Migration\V6_7\Migration1756068712FixOrderAddressLastNameLength;
+
+/**
+ * @internal
+ */
+#[CoversClass(Migration1756068709FixCustomerAddressFirstNameLength::class)]
+#[CoversClass(Migration1756068710FixCustomerAddressLastNameLength::class)]
+#[CoversClass(Migration1756068711FixOrderAddressFirstNameLength::class)]
+#[CoversClass(Migration1756068712FixOrderAddressLastNameLength::class)]
+class Migration1756068709FixAddressNameFieldLengthsTest extends TestCase
```

Comment:
when we split the original class we should split the test classes as well right?